Src/Coravel/Queuing/HostedService/QueuingHost.cs

```diff
@@ -15,14 +16,19 @@ internal class QueuingHost : IHostedService, IDisposable
         private SemaphoreSlim _signal = new SemaphoreSlim(0);
         private Timer _timer;
         private Queue _queue;
-        public QueuingHost(IQueue queue)
+        private IConfiguration _configuration;
+
+        public QueuingHost(IQueue queue, IConfiguration configuration)
         {
+            this._configuration = configuration;
             this._queue = queue as Queue;
         }
 
         public Task StartAsync(CancellationToken cancellationToken)
         {
-            this._timer = new Timer((state) => this._signal.Release(), null, TimeSpan.Zero, TimeSpan.FromSeconds(30));
+            int consummationDelay = this._configuration.GetValue<int>("Coravel:Queue:ConsummationDelay", 30);
```

Comment:
A possible alternative (which could be moved to a private method):

```c#
bool couldParseDelay = int.TryParse(this._configuration.GetSection("Coravel:Queue:ConsummationDelay").Value, out var parsedDelay);

int consummationDelay = couldParseDelay ? parsedDelay : 30; 
```

Otherwise everything looks good 👍